lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -426,7 +426,7 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
                                 other {# stops}}",
             num_informed_platforms: length(informed_platforms)
           ),
-        location: %{full: "mbta.com/alerts", abbrev: "mbta.com/alerts"}
+        location: %{full: "mbta.com/status", abbrev: "mbta.com/status"}
```

Comment:
At least within this module, we should put this in a module attribute so it's not repeated 4 times.